src/translations/en.json

```
@@ -2308,6 +2308,12 @@
             "unknown": "unknown",
             "wakeup_interval": "Wakeup Interval"
           },
+          "migration": {
+            "ozw": {
+              "header": "Migrate to OpenZWave",
+              "introduction": "This wizard will help you migrate from the legacy Z-Wave component to the OpenZWave integration that is currently in beta."
```

```suggestion
              "introduction": "This wizard will help you migrate from the legacy Z-Wave integration to the OpenZWave integration that is currently in beta."
```
Should we mention the 1.4 version here as well for the legacy integration?